_posts/2022-11-20-point-in-time.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: post
+title:  "Launch highlight: Paginate with Point in Time"
+authors:
+- gbh
+- snistala
+- abandeji
+- movvaam
+- dhruvdas
+- bukhtawa
+- kolchfa
+date: 2022-11-20 00:00:00 -0700
+categories:
+ - community
+
+excerpt: New for version 2.4, Point in Time (PIT) is the preferred pagination method for OpenSearch. While OpenSearch still supports other ways to paginate results, PIT search proves superior in capabilities and performance because it is not bound to a query and it supports consistent pagination going forward and backward.
+---
+
+New for [version 2.4](https://opensearch.org/blog/releases/2022/11/opensearch-2-4-is-available-today/), Point in Time (PIT) is the preferred pagination method for OpenSearch. While OpenSearch still supports other ways to paginate results, PIT search proves superior in capabilities and performance because it is not bound to a query and it supports consistent pagination going forward and backward.
```

Comment:
```suggestion
New for [version 2.4](https://opensearch.org/blog/releases/2022/11/opensearch-2-4-is-available-today/), Point in Time (PIT) is the preferred pagination method for OpenSearch. While OpenSearch still supports other ways of paginating results, PIT search provides superior capabilities and performance because it is not bound to a query and supports consistent pagination going both forward and backward.
```